### DISCUSSION TITLE
No translation - Pas de traduction

### DISCUSSION THREAD

**Discussion by raphaelmarie** [DivinumOfficium/divinum-officium#3697] [Developers corner]
Hello,

Although the translation is in French, the Martyrology is still in English. How can I change this or what is the method?

Thank you

Bonjour,

Bien que la traduction soit faite en langue française, au niveau du Martyrologe, ce texte demeure en anglais. Comment changer cela ou quelle est la méthode ?

Merci

[Quad6-4]  	
Le Jeudi Saint, lorsque le Christ Jésus, la veille de sa crucifixion pour notre salut, donna à ses disciples les mystères de son Corps et de son Sang, à célébrer par eux.

**Comment by rl1234567890** [comment]
Sorry, I don't speak French. You should provide a translation of the Mobile.txt files that are found in 
./Martyrologium1960/Mobile.txt
./Martyrologium1570/Mobile.txt
./Martyrologium/Mobile.txt
./Martyrologium1955R/Mobile.txt

**Reply by rl1234567890** [reply]
ACtually the English has only ./Martyrologium/Mobile.txt

**Comment by APMarcello3** [comment]
Le fichier "Martyrologium/Mobile texte" a été renommé Mobile.txt.
